content/blog/newsletter/2024-04-05-newsletter.md

```diff
@@ -0,0 +1,236 @@
+---
+title: The XMPP Newsletter March 2024
+date: 2024-04-07
+categories: ['Newsletter']
+---
+
+{{< figure src="images/newsletter/xmpp_newsletter_banner.jpg" caption="XMPP Newsletter Banner" >}}
+
+# The 50<sup>th</sup> release of the XMPP Newsletter!
+
+This is the 50th release of the XMPP Newsletter since it started in February 2019.
+We think it is worth to celebrate this achievement and say thanks to all the contributors as well as all our readers!
+Back then at the Summit at Brussels JC Brand, Nicolas Vérité (Nyco) and Severino Ferrer (S0ul) proposed and initialed the XMPP Newsletter and since then we had almost every month a release of the news in the XMPP universe.
```

Comment:
```suggestion
Back at the Summit in Brussels, JC Brand, Nicolas Vérité (Nyco) and Severino Ferrer (S0ul) proposed and initiated the XMPP Newsletter.
Since then almost every month there has been a release full of news from the XMPP universe.
```